### PR TITLE
wip

### PR DESCRIPTION
**Pull request by xh3b4sd**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/internal/cloudconfig/fixture/f001/load.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+		if tls.Key, ok = secret.Data["key"]; !ok {
+			return microerror.Maskf(invalidSecretError, "%q key missing", "key")
+		}
+	}
+
+	return nil
+}
+
+func mustLoad(f string) []byte {
+	p := filepath.Join("/Users/xh3b4sd/go/src/github.com/giantswarm/aws-operator/service/internal/cloudconfig/fixture/f001", f)
```
Comment (calvix) [inline service/internal/cloudconfig/fixture/f001/load.go:172]: this looks like accident static linking

**Comment by calvix** [comment]
Can you give a short description of this?What is important to check what is not? I assume its  unittest for k8scc rendering

**Comment by xh3b4sd** [comment]
This is my playground for the k8scc rendering. I cannot reproduce the rendering issues and just did some quick and dirty PoC to get results quicker. When we want this to have golden file tests for the cloud config rendering again we have to polish this more and get it in proper shape. Earlier I just pushed what I had to debug another issue with HA Masters.

**Comment by xh3b4sd** [comment]
No prio on that right now.

**Pull request closed without merging**